js/main.js

```diff
@@ -61,12 +61,12 @@ const getRandomElements = function (array = [], count = 0) {
 
 const getIntWithLeadingZeros = (number) => number < 10 ? `0${number}` : number;
 
-const generateAnnouncements = function (count, minLat, maxLat, minLng, maxLng, locationAccuracy, minPrice, maxPrice, houseTypes, minRoom, maxRoom, minGuest, maxGuest, checkInOutTimes, houseFeatures, photos) {
+const generateAnnouncements = (count) => {
```

Comment:
Старайся всегда задавать значения по умолчанию)))
```suggestion
const generateAnnouncements = (count = 0) => {
```

*Поправь в рамках след. задания просто